census_view.hpp

```diff
@@ -134,6 +134,14 @@ class CensusView final
     wxDataViewCtrl* list_window_;
     wxObjectDataPtr<CensusViewDataViewModel> list_model_;
 
+    struct header_cache
+    {
+        std::string name;
+        bool        show;
```

Comment:
We have `oenum_visibility` now and it would be better to use it here, i.e. make this `oenum_visibility visibility{oe_shown};` (I've added an initializer because there doesn't seem to be any good reason to not initialize this field...).